styleguide/components/31-molecules/building-blocks/form-steps/_form-steps.scss

```diff
@@ -120,19 +86,32 @@
     }
 
     &.progress-previous {
-      .count i {
-        margin-right: 0;
-        font-size: .8rem;
-        line-height: 1.6rem;
+      &.progress-active .count {
+        @include theme('border-color', 'color-secondary', 'form-steps-active-prefix-border-color');
+        @include theme('background', 'color-secondary', 'form-steps-active-prefix-background');
+      }
+
+      &.progress-active a.indicator {
+        @include theme('color', 'color-secondary', 'form-steps-active-step-label');
       }
 
       .count {
+        @include theme('background-color', 'color-secondary', 'form-steps-previous-step-background');
+
         border-radius: 100%;
         text-align: center;
         text-indent: 0;
+
+        i {
+          margin-right: 0;
+          font-size: .8rem;
+          line-height: 1.6rem;
```

Comment:
I always find rem on line-height a bit redundant.
w3c: A number that will be multiplied with the current font-size to set the line height

So just '2' will do.

Reply:
Also not in the scope of this PR :)